ops/terraform/modules/mgmt_stateful/main.tf

```diff
@@ -5,16 +5,15 @@
 #
 
 locals {
-  azs = ["us-east-1a", "us-east-1b", "us-east-1c"]
-  env_config = {env=var.env_config.env, tags=var.env_config.tags, vpc_id=data.aws_vpc.main.id, zone_id=aws_route53_zone.local_zone.id }
+  env_config = {env=var.env_config.env, tags=var.env_config.tags, vpc_id=data.aws_vpc.main.id, zone_id=aws_route53_zone.local_zone.id, azs=var.env_config.azs}
 }
 
 # VPC
 #
 data "aws_vpc" "main" {
   filter {
     name = "tag:Name"
-    values = ["bfd-${var.env_config.env}-vpc"]
+    values = ["bfd-mgmt-vpc"]
```

Comment:
I feel like switching to hard coded values is a step backwards, if it is necessary to achieve a short term goal we should create a tech debt ticket to re-variable-ize in the future.

Reply:
It's only necessary for the shared services that mgmt and mgmt-test will rely on. Since there are no other VPC, SG's, etc outside of the mgmt environment, these have to be hard coded so that the env variable can be used to name other resources that will conflict. I'd be happy to elaborate on this further if you have questions. See the other resources and it might make since.